reference/04-Patterns/synced-patterns.md

```diff
@@ -0,0 +1,152 @@
+</div>
+<!-- /wp:button -->
+
+</div>
+<!-- /wp:buttons -->
+
+</div>
+<!-- /wp:group -->
+```
+
+### 2. Create a block template part to house your pattern
+
+In your themes `parts` directory, create a new file containing your pattern.  Perhaps you only have the one pattern you wish to sync in which case this file could be called `call-to-action.html`, though depending on preference it might be nicer to use a single file to add **all** Synced Patterns, in which case `synced-patterns.html` would be more semantic.
+
+```html title="parts/synced-patterns.html"
+<!-- wp:pattern {"slug":"tenup-theme/call-to-action"} /-->
+
+<!-- wp:pattern {"slug":"tenup-theme/a-second-pattern-to-be-synced"} /-->
+
+```
+
+[For more information on Block Template Parts, refer to this section of the Gutenberg Best Practices](../../reference/Themes/block-template-parts).
+
+:::info
+
+### Why are we doing this?
+
+In the first step, we marked our pattern as hidden and disabled it from being inserted manually.  By doing so, we are ensuring that editors will never have access to the original, unsynced pattern and cannot use our themes component in posts or pages until it has been marked as Synced and added to the database.
+:::
+
+### 3. Mark your pattern as Synced
+
+Now that we have a block template part in place, navigate to the Site Editor in the WordPress admin and select the Synced Patterns template part.  It is here that we will [follow the steps outlined above to sync](#creating-a-synced-pattern).
+
+:::caution
+This step needs to be taken in all environments.
+:::
+
+Your pattern is now Synced and has been added to the database. It is ready to use throughout your site.
+
+### 4. Updating across environments
+
+Should we wish to change the styles or text used, we will still have to manually update our Synced Pattern in each enviroment.
+
+One approach might be to:
+
+1. Open each enviroment in a separate browser tab and navigate to your themes Patterns directory under the dashboard Appearance tab. Choose your Synced Pattern.
+2. In one browser tab, make your updates as desired and hit **Save**. Then select **Copy all blocks** from the Options settings in the upper right of the screen.
+3. In the other browser tabs, overwrite the pattern by copying the updated one into the editor.
+4. Optionally, you may want to also copy our updates back into the `patterns/call-to-action.php` file to keep that up to date.
+
+:::caution
+It is important to note this last step is a fairly major difference in traditional theme authoring.  Typically you'd rewrite your component locally first, push the updated codebase, *then* see your changes in the other environments.  Here, we must make our updates in all environments first, and essentially work backwards to update a Synced Pattern in our theme.
+
+This is another reason trying to maintain a Synced Pattern as a theme author should come with special consideration as  mentioned above.
```

Comment:
I think we can be even more opinionated than that. 
﻿
I would say for these reasons we don't recommend the usage of synced patterns for most sites today. 
﻿
And then link to the relevant GitHub issue for allowing the definition of synced theme patterns

Reply:
Done!